Assets/PlayerMovement.cs

```diff
@@ -0,0 +1,43 @@
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+public class PlayerMovement : MonoBehaviour
+{
+    [SerializeField] public float moveSpeed = 5f;
+    private Rigidbody2D rb;
+    private Vector2 movement;
```

Comment:
Does this need to be initialized?
If Update() calls before OnMove() fires will this throw a null reference expectation?